src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -42,6 +42,17 @@ export type WDSCloneStatusResponse = {
   updated: string;
 };
 
+export interface WDSJob {
+  created: string;
+  errorMessage: string | null;
+  input: null;
+  jobId: string;
```

Comment:
do we have a concept of/type for UUID in this codebase? jobId will be a UUID - it's also fine to just leave it as a string here.

Reply:
We don't. We could add a "branded type" for WDS job IDs though.
https://egghead.io/blog/using-branded-types-in-typescript

Reply:
Definitely not necessary for this PR, just something to think about for the future.

Reply:
I think the use for such a type will become more apparent in the changes for https://broadworkbench.atlassian.net/browse/AJ-1431.